packages/legend-application-studio/src/stores/editor/editor-state/element-editor-state/connection/DatabaseBuilderState.ts

```diff
@@ -799,61 +797,77 @@ export class DatabaseSchemaExplorerState {
     }
   }
 
-  *updateDatabase(): GeneratorFn<void> {
+  // this method just updates database
+  *updateDatabase(): GeneratorFn<Database> {
+    this.isUpdatingDatabase = true;
```

Comment:
again why are we removing this ?

Reply:
I didn't remove this method.  I just separate part of it to a new method and rename the original one..